Python/math/Strong_Number.py

```diff
@@ -0,0 +1,43 @@
+"""
+Basically a strong number is number in which sum of factorial of each digit
+of the number is equal to that number
+EX : Consider 145=1! + 4! + 5! = 1 + 24 + 120 = 145
+Here input a number
+We initialize sum to 0 and assign the input number to a temporary variable
+In side while loop --> (we find sum of factorial of each digit of the number)
+In side factorial function --> (we calculate the factorial of each digit and return it)
+In the end if number and the sum obtained is equal then it is a Strong number
+otherwise not
+"""
+
+
```

Comment:
Remove extra one line space.

Reply:
@ankitaggarwal23 In first commit I gave like that but in DeepSource Python It gave me There must be 2 line spaces btw code and func  I add 1 extra line space btw them